### ISSUE TITLE
sampling issue

### ISSUE THREAD

**Issue by signalstop** [gazingstars123/Anima-Standalone-Trainer#3]
[model_arguments]
dit_path = "V:\\AI\\Tools\\Anima-Standalone\\models\\anima-preview.safetensors"
qwen3_path = "V:\\AI\\Tools\\Anima-Standalone\\models\\qwen_3_06b_base.safetensors"
vae_path = "V:\\AI\\Tools\\Anima-Standalone\\models\\qwen_image_vae.safetensors"

[dataset_arguments]
dataset_config = "V:\\AI\\Tools\\Anima-Standalone\\training-ui\\jobs\\wfare\\dataset.toml"
cache_latents_to_disk = true
cache_text_encoder_outputs_to_disk = false

[training_arguments]
output_name = "wfare"
save_model_as = "safetensors"
max_train_steps = 2_000
save_every_n_steps = 200
sample_every_n_steps = 100
log_with = "tensorboard"
learning_rate = 0.0001
text_encoder_lr = 0.00005
optimizer_type = "AdamW8bit"
lr_scheduler = "cosine"
lr_warmup_steps = 100
mixed_precision = "bf16"
max_data_loader_n_workers = 4
gradient_accumulation_steps = 1
max_grad_norm = 1
gradient_checkpointing = true
flash_attn = false
lowram = false
blocks_to_swap = 0
persistent_data_loader_workers = true
seed = 42
vae_batch_size = 1
output_dir = "V:\\AI\\Tools\\Anima-Standalone\\training-ui\\jobs\\wfare\\output"
logging_dir = "V:\\AI\\Tools\\Anima-Standalone\\training-ui\\jobs\\wfare\\logs"
save_state = true
save_last_n_steps_state = 1
save_last_n_epochs_state = 1

[sample_arguments]
sample_prompts = "V:\\AI\\Tools\\Anima-Standalone\\training-ui\\jobs\\wfare\\sample_prompts.txt"
sample_every_n_epochs = 1

[network_arguments]
network_module = "networks.lora_anima"
network_dim = 16
network_alpha = 16
network_train_unet_only = true

[anima_arguments]
timestep_sample_method = "logit_normal"
discrete_flow_shift = 3
weighting_scheme = "logit_normal"

---------------------------------

sample_arguments doesn't seem to update when using steps or when complety unchecking "enable sampling". I think it interprets the 1 epoch as a step and generates samples every step.

**Comment by signalstop** [comment]
[sample_arguments]
sample_prompts = "V:\\AI\\Tools\\Anima-Standalone\\training-ui\\jobs\\wfare\\sample_prompts.txt"
sample_every_n_epochs = 1

I manually changed the toml to sample_every_n_steps = 100 but it changed back on its own to sample_every_n_epochs = 1 without me saving anything when I started the job.

**Comment by signalstop** [comment]
<img width="826" height="223" alt="Image" src="https://github.com/user-attachments/assets/8003bbb1-08ce-49e9-a0c0-3c76f5a93df5" />

Any idea what this is?  I'm 1000 steps in.  I get that message after each sample runs.

**Comment by gazingstars123** [comment]
It's completely harmless so you can safely ignore it
